src/gui/editors/AutomationEditor.cpp

```diff
@@ -1261,6 +1266,48 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 			p.drawLine( x, grid_bottom, x, x_line_end );
 		}
 
+		// draw ghost notes
+		// const NoteVector* currentMidiNotes = &(getGUI()->pianoRoll()->currentMidiClip()->notes());
```

Comment:
I don't understand this comment, is this some commented out left over from previous versions of the PR?

Reply:
Yes sorry, I wasn't expecting such a quick review. I was testing using the active midi clip as the ghost notes, but I decided against it. Already removed.